### ISSUE TITLE
Misleading variable names: Coal 

### ISSUE THREAD

**Issue by felixfrischmuth** [openENTRANCE/nomenclature#52]
When checking the nomenclature with our model output, I got two warnings. This leads me to this issue.
In the nomenclature (nomenclature/nomenclature/definitions/variable/technology/technologies.yaml) is the differentiation between "Coal” and "Lignite". I would suggest differing between „Hard Coal“ and „Lignite“, and maybe “Coal” for sum them up.
In nomenclature (nomenclature/nomenclature/definitions/variable/energy/...) there is only the variable "Coal". It should also include "Lignite" and "Hard coal", e.g. Secondary Energy|Electricity|Lignite. 
If you think this too, I can do a pull request?

**Comment by tburandt** [comment]
I support the suggestion of @felixfrischmuth to add the different subtypes of coal and provide a uniform way of handling it. So, I would propose to also add "Coal" for summing up the secondary energy variables as well as the technology variables.

E.g.:

`Secondary Energy|Electricity|Coal`
`Secondary Energy|Electricity|Coal|Lignite`
`Secondary Energy|Electricity|Coal|Hard coal`

and

`Capacity|Electricity|Coal`
`Capacity|Electricity|Coal|Lignite`
`Capacity|Electricity|Coal|Hard coal`

but of course for all related variables

**Comment by arght** [comment]
In that regard we may also add bituminous and sub-bituminous coal if anybody is willing to use them

**Comment by sandrinecharousset** [comment]
> I support the suggestion of @felixfrischmuth to add the different subtypes of coal and provide a uniform way of handling it. So, I would propose to also add "Coal" for summing up the secondary energy variables as well as the technology variables.
> 
> E.g.:
> 
> `Secondary Energy|Electricity|Coal`
> `Secondary Energy|Electricity|Coal|Lignite`
> `Secondary Energy|Electricity|Coal|Hard coal`
> 
> and
> 
> `Capacity|Electricity|Coal`
> `Capacity|Electricity|Coal|Lignite`
> `Capacity|Electricity|Coal|Hard coal`
> 
> but of course for all related variables

I agree :-)

**Comment by danielhuppmann** [comment]
I also agree with this suggestion (in the sub-category format of `Coal` as suggested by @tburandt)!

@felixfrischmuth if you need this urgently, please go ahead and start a PR - otherwise, it might make more sense to wait until I make a PR to implement a generic <fuel> identifier (see #29), so that you don't have to add the coal disagggregation for every fuel.
